test/distrib/cpp/run_distrib_test.sh

```diff
@@ -30,7 +30,13 @@
 
 set -ex
 
-git clone --recursive $EXTERNAL_GIT_ROOT
+git clone $EXTERNAL_GIT_ROOT
```

Comment:
when $EXTERNAL_GIT_ROOT is used as the source repo, you need to use it (instead of /var/local/jenkins/grpc)